libyul/AsmAnalysis.h

```diff
@@ -84,7 +86,8 @@ class AsmAnalyzer
 	static AsmAnalysisInfo analyzeStrictAssertCorrect(
 		Dialect const& _dialect,
 		Block const& _astRoot,
-		std::set<std::string> const& _qualifiedDataNames
+		std::set<std::string> const& _qualifiedDataNames,
+		std::string const& _objectName
```

Comment:
Would be best to keep the order of args consistent. And since `_qualifiedDataNames` can have a default (it could have one here too), it should go second:

```suggestion
		std::string const& _objectName,
		std::set<std::string> const& _qualifiedDataNames
```